src/store/bond/actions.test.js

```diff
@@ -0,0 +1,123 @@
+import { binanceChainMock, ethereumChainMock } from '../../utils/chain.mock'
+import { TEST_TOKEN, TEST_WALLET, TEST_POOL } from '../../utils/web3'
+import {
+  getBondListed,
+  getBondListedAsset,
+  getBondDepositEstimate,
+  getBondClaimable,
+  getBondMemberDetails,
+  getBondSpartaRemaining,
+  getBondBurnReady,
+  getBondProposalCount,
+  getBondProposal,
+  getBondProposals,
+  getBondCoolOffPeriod,
+} from './actions'
+import * as Types from './types'
+
+window.BinanceChain = binanceChainMock
+window.ethereum = ethereumChainMock
+
+describe('Bond actions', () => {
+  let dispatchMock
+
+  beforeEach(() => {
+    dispatchMock = jest.fn()
+  })
+
+  afterEach(() => {
+    dispatchMock.mockRestore()
+  })
+
+  test('should get bond listed', async () => {
+    await getBondListed()(dispatchMock)
+    console.log(dispatchMock.mock.calls[1][0])
+    expect(dispatchMock.mock.calls[1][0].payload).toStrictEqual([
+      '0x0000000000000000000000000000000000000000',
+      '0x90c92451e0D2e439D2ED15bd2C6Ba3B31d42571B',
+      '0xeD24FC36d5Ee211Ea25A80239Fb8C4Cfd80f12Ee',
+      '0x07dc3a8cD1B54CDFd55d223Ca15863dcA6B70C1A',
+      '0xb0bbC81b0769A57079E01FE1333E060e7Cd438Aa',
+    ])
+    expect(dispatchMock.mock.calls[1][0].type).toBe(Types.GET_BOND_LISTED)
+  })
+
+  test('should get bond listed asset', async () => {
+    await getBondListedAsset(TEST_TOKEN)(dispatchMock)
+    console.log(dispatchMock.mock.calls[1][0])
+    expect(dispatchMock.mock.calls[1][0].payload).toBe(false)
+    expect(dispatchMock.mock.calls[1][0].type).toBe(Types.GET_BOND_LISTED_ASSET)
+  })
+
+  test('should get bond deposit estimate', async () => {
+    await getBondDepositEstimate(TEST_TOKEN, 1000)(dispatchMock)
+    console.log(dispatchMock.mock.calls[1][0])
+    expect(dispatchMock.mock.calls[1][0].payload).not.toBeUndefined()
+    expect(dispatchMock.mock.calls[1][0].type).toBe(
+      Types.GET_BOND_DEPOSIT_ESTIMATED,
+    )
+  })
+
+  test('should get bond claimable', async () => {
+    await getBondClaimable(TEST_WALLET, TEST_POOL, TEST_TOKEN)(dispatchMock)
+    console.log(dispatchMock.mock.calls[1][0])
+    expect(dispatchMock.mock.calls[1][0].payload).not.toBeUndefined()
+    expect(dispatchMock.mock.calls[1][0].type).toBe(Types.GET_BOND_CLAIMABLE)
+  })
+
+  test('should get bond member details', async () => {
+    await getBondMemberDetails(TEST_WALLET, TEST_POOL, TEST_TOKEN)(dispatchMock)
+    console.log(dispatchMock.mock.calls[1][0])
+    expect(dispatchMock.mock.calls[1][0].payload).not.toBeUndefined()
+    expect(dispatchMock.mock.calls[1][0].type).toBe(
+      Types.GET_BOND_MEMBER_DETAILS,
+    )
+  })
```

Comment:
Both of these; swapped 'TEST_WALLET' for 'BOND_ADDR'
and swapped 'TEST_POOL' for 'TEST_WALLET'

Reply:
Oh and 'TEST_TOKEN' swapped for 'BNB_ADDR' as BNB is listed in bond instead of WBNB